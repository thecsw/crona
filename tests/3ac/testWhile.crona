main : void () {
    while (true) {
        write 1;
    }
}