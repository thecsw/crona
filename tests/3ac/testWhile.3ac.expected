[BEGIN GLOBALS]
[END GLOBALS]
[BEGIN main LOCALS]
[END main LOCALS]
main:       enter main
lbl_1:      nop
            IFZ 1 GOTO lbl_2
            WRITE 1
            goto lbl_1
lbl_2:      nop
lbl_0:      leave main